CHANGELOG.rst

```diff
@@ -7,6 +7,7 @@ Unreleased
 Updated
 -------
 - Preferring Bearer token over user token when configuring Connection session
+- Using json-api runs endpoint to fetch project runs, avoid possible timeouts, and improve efficiency
```

Comment:
Let's call this out as a breaking change - the run resource returned has been pared down. Do you know if this gets consumed when we publish to pypi?